.gitignore

```diff
@@ -15,5 +15,4 @@ webpack-stats.json
 *.egg-info
 .DS_STORE
 CACHE
-.tsconfig-paths.json
-.frontend-configuration-settings.json
+frontend_configuration/
```

Comment:
I'm not sure if we can do a straight updateproject here, given that this project supports both 7.6 and 8. Let's wait for Christopher's review

Reply:
Yeah, this is fine IMO as it goes hand-in-hand with the changes in apps.py.

Reply:
But we'll likely want to git ignore both locations if we support both versions

Reply:
nvm, I see now that these files *aren't* generated here when another 7.6 project depending on it is run. so I think this change is fine